docs/source/en/tasks/question_answering.mdx

```diff
@@ -370,7 +370,7 @@ Pass your inputs to the model and return the `logits`:
 
 ```py
 >>> from transformers import AutoModelForQuestionAnswering
```

Comment:
nit: sticking with pep8, third party modules should be imported first. 

Annoyingly, github won't let you make suggestions on lines which include deletions, but you'd also need to delete the `import torch` line below

```suggestion
>>> import torch
>>> from transformers import AutoModelForQuestionAnswering
```